content/it/partners/_index.html

```diff
@@ -9,10 +9,101 @@
 <section id="users">
     <main>
         <h5>Kubernetes collabora con i partner per creare una base di codice forte e vivace che supporti uno spettro di piattaforme complementari.</h5>
+        <div class="col-container">
+          <div class="col-nav">
+            <center>
+              <h5---
+title: Partners
+bigheader: I Partners di Kubernetes
+abstract: La Crescita dell'ecosistema Kubernetes.
+class: gridPage
+cid: partners
+---
+<meta charset="utf-8" />
+
+<section id="users">
+    <main>
+        <h5>Kubernetes lavora in collaborazione con diversi partner al fine di creare un codice dalla base forte e dinamica che supporti uno spettro di piattaforme complementari.</h5>
```

Comment:
Same as before
> This is the literal translation of the English sentence but honestly, it does not sound great in Italian. I personally do not understand what this means (grammatically is correct though). I think we should rephrase this to something else.

Reply:
Hi @mattiaperi, any idea how we can rephrase this sentence?
Thanks in advance for your help

Reply:
Much better overall, I suggest this:

```
Kubernetes collabora con vari partner per creare un sorgente di codice robusto e al tempo stesso dinamico, in grado di supportare una moltitudine di piattaforme.
```
-> 
```
Kubernetes collabora con vari partner per creare codice sorgente che sia robusto e al tempo stesso dinamico, in grado di supportare una moltitudine di piattaforme.
```